utilities/jenkins/outputs.tf

```diff
@@ -0,0 +1 @@
+
```

Comment:
Should we just delete this file?

Reply:
i left it because we'll probably add a few things in there eventually like the private address for the master as we work through using it "for real." if it's still empty once we're all the way through transitioning to it next week, then we'll garbage collect it.